contracts/WstETHApi3ReaderProxyV1.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.27;
+
+import "@api3/contracts/interfaces/IApi3ReaderProxy.sol";
+import "./IWstETH.sol";
+
+/// @title An immutable proxy contract that reads the price of wstETH/stETH
+/// directly from the WstETH contract on Ethereum.
+/// @dev This contract implements only the IApi3ReaderProxy and not the
+/// AggregatorV2V3Interface which is usually implemented with Api3 proxies. The
+/// user of this contract needs to be aware of this and only use this contract
+/// where the IApi3ReaderProxy interface is expected.
+contract WstETHApi3ReaderProxyV1 is IApi3ReaderProxy {
```

Comment:
We'll need to deploy and verify this contract on etherscan. What is the best way to do that?

Reply:
Current hardhat version uses [ignition plugin](https://hardhat.org/ignition/docs/getting-started#overview) for deploying contract but I've had some issues with it in the past like trying to get the deployment block for a contract, etc. So on the other repos we usually just rely on [hardhat-deploy plugin](https://www.npmjs.com/package/hardhat-deploy) to handle deployments via deployment scripts. For verifying the contracts we can use [hardhat-verify plugin](https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-verify) and run it in a deployment script like [@api3/contracts does it](https://github.com/api3dao/contracts/blob/main/deploy/3_verify.ts).

Reply:
Yeah, let's do the HH deploy plugin. 

I don't think we need the verify plugin in the repo, but I'll need this for the deployment. Let's quickly sync on that on Slack.